README.md

```diff
@@ -3,7 +3,7 @@
 Repository with various files to install CircleCI's runner on Kubernetes via Helm chart.
 
 ## Prerequisites
-- You must be on our [Scale Plan](https://circleci.com/pricing/) or sign up for a trial. [Reach out to our sales team](https://circleci.com/contact-us/?cloud) to ask about both.
+- You must be on one of our [Performance or Scale plans](https://circleci.com/pricing/), or sign up for a trial. [Reach out to our sales team](https://circleci.com/contact-us/?cloud) to ask about both.
```

Comment:
I think runner is available on all plans now, aside from custom?

Reply:
Done. Just removed the plan prerequisite.